### ISSUE TITLE
Scema.orgno

### ISSUE THREAD

**Issue by nichesuch** [Code-for-SAKE/Sakepedia-Nuxt#138]

**Comment by erikka-22** [comment]
GraphDBを立てて、MongoDBとの連携プラグインを用いて連携して、GraphDBをsparqlエンドポイントとして用いるというのが一つの方法としてあるようです！

https://graphdb.ontotext.com/documentation/10.0/mongodb-graphdb-connector.html

**Comment by erikka-22** [comment]
検索エンジン対応であれば、HTMLタグにitemscope, itemtype, itemprop を使ってSchema.orgの語彙をマークアップするのも意味があるようですが、SakepediaはSEOを意識して...とかではないですよね。

https://developer.mozilla.org/ja/docs/Web/HTML/Global_attributes/itemscope

**Comment by nichesuch** [comment]
> SakepediaはSEOを意識して...とかではないですよね。

そうですね。やるとしても今のURL.jsonldとかでアクセスしたら、Schema.orgのデータが取れるって感じが良いかなと思います。

**Comment by nichesuch** [comment]
そこをGoogleさんにクロールしてもらって、リッチスニペットで表示されたら面白そう。
